setup.py

```diff
@@ -4,7 +4,6 @@
 
 extras_require = {
     'test': [
-        'unittest2',
     ],
 }
```

Comment:
The `test` extra is quite empty now. We could remove it altogether. You would need to change the `setup()` method too.

Reply:
😄 good point!